cobalt/browser/BUILD.gn

```diff
@@ -114,6 +113,25 @@ action("embed_polyfilled_javascript") {
   ]
 }
 
+source_set("global_features") {
+  # TODO(cobalt, b/375655377): remove testonly declaration, needed because of
+  # being depended-on by :browser above.
+  testonly = true
+
+  sources = [
+    "global_features.cc",
+    "global_features.h",
+  ]
+  deps = [
+    ":metrics",
+    "//components/metrics",
+    "//components/metrics:test_support",
```

Comment:
[This](https://source.corp.google.com/h/lbshell-internal/cobalt_src/+/main:components/metrics/BUILD.gn;l=279-294;drc=210cfa12aefea48375aa1b81b1929aa1bca2313b) shouldn't be needed for us.

Reply:
Done